docs/design/autoware-interfaces/components/sensing.md

```diff
@@ -0,0 +1,52 @@
+# Sensing
+
+![Node diagram](images/Sensing-Bus-ODD-Architecture.drawio.svg)
+
+## Inputs
+
+### The sensing module contains the driver for the sensor, so there is no input
+
+## Output
+
+### Camera image
+
+Image data from camera. Used by the Perception.
+
+- [sensor_msgs/Image](https://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/Image.html)
+
+### Radar tracks
+
+Tracks from radar. Used by the Perception.
+
+- [radar_msgs/RadarTracks](https://docs.ros.org/en/noetic/api/radar_msgs/html/msg/RadarTracks.html)
```

Comment:
I think we are still open with the message type. Let' make it TBD for now.

Reply:
Related discussion: https://github.com/orgs/autowarefoundation/discussions/2531

Reply:
```suggestion
- [radar_msgs/RadarTracks](https://github.com/ros-perception/radar_msgs/blob/ros2/msg/RadarTracks.msg)
```

[I will try to end the discussion](https://github.com/orgs/autowarefoundation/discussions/2531#discussioncomment-5977058), I think it's OK to use https://github.com/ros-perception/radar_msgs/tree/ros2/msg in the pipeline.